### DISCUSSION TITLE
extract email data

### DISCUSSION THREAD

**Discussion by test2a** [postalserver/postal#1794] [Help with using Postal] [answered]
hi. i am considering using postal for a project but i am not sure if it can fully help.
my problem. 
i expect to receive transactional emails on clients emails who have been instructed to forward them to clientA@ab.com 
postal is supposed to take the email , "extract" one time password from the email body and then pass that otp to clientA http request which will be used in automation at client side software.

https://www.mailslurp.com/guides/extract-email-content/
mailslurp seems to have this but i could not find postal documentation on it. it is possible i missed something, in which case i would be glad if someone can point me in the right direction..

**Comment by willpower232** [comment, marked as answer]
Postal is mostly designed to send emails. It can receive them but it would have to pass them through to your own code or separate system to process out the content.

**Reply by test2a** [reply]
alright. so postal might not be so useful for my work. thanks for answering anyways. good luck with your project
